installer/installer.go

```diff
@@ -30,7 +30,7 @@ type UInstaller interface {
 	EnableUpdatedPartition() error
 }
 
-func Install(art io.ReadCloser, dt string, device UInstaller) error {
+func Install(art io.ReadCloser, dt string, key []byte, device UInstaller) error {
```

Comment:
I think this should be squashed with the previous commit. I think the previous commit won't compile because it makes a reference to this `key` variable.